video-stream/app.py

```diff
@@ -21,12 +21,12 @@ def process_packets(self, packet: DetectionPacket):
 
             current_time = time.time()
             # arbitrary condition for sending image events to RobotHub
-            if current_time - self.last_image_event_upload > CONFIGURATION["image_event_upload_interval"]:
+            if current_time - self.last_image_event_upload > CONFIGURATION["image_event_upload_interval"] * 60:
                 if detection.label == 'person':
                     self.last_image_event_upload = current_time
                     send_image_event(image=packet.frame, title='Person detected')
             # arbitrary condition for sending video events to RobotHub
-            if current_time - self.last_video_event_upload > CONFIGURATION["video_event_upload_interval"]:
+            if current_time - self.last_video_event_upload > CONFIGURATION["video_event_upload_interval"] * 60:
```

Comment:
maybe even using 
`self.last_video_event_upload_seconds`
and
`current_time_seconds`
would make the code very much readable. Would not leave any room for doubts.